assets/src/editor/events/tickets/editor-ticket/price-calculator/price-modifier-row/price-amount-input.js

```diff
@@ -83,4 +84,13 @@ const PriceAmountInput = ( {
 	] );
 };
 
+PriceAmountInput.propTypes = {
+	prefix: PropTypes.string.isRequired,
+	values: PropTypes.object.isRequired,
+	price: PropTypes.object.isRequired,
+	priceType: PropTypes.object.isRequired,
+	ticket: PropTypes.object.isRequired,
```

Comment:
ahh dude... so sry... looks like the `ticket` prop was completely removed from the component... my bad AGAIN!